generate/generate.go

```diff
@@ -307,33 +311,37 @@ func generatedHCLOutdatedFiles(
 		Str("stackpath", stackpath).
 		Logger()
 
-	logger.Trace().Msg("Checking for outdated exported terraform code on stack.")
+	logger.Trace().Msg("Checking for outdated generated_hcl code on stack.")
 
-	loadedStackTf, err := genhcl.Load(root, stackMeta, globals)
+	stackHCLs, err := genhcl.Load(root, stackMeta, globals)
 	if err != nil {
 		return nil, err
 	}
 
-	logger.Trace().Msg("Loaded exported terraform code, checking")
+	logger.Trace().Msg("Loaded generated_hcl code, checking")
 
 	outdated := []string{}
 
-	for filename, genHCL := range loadedStackTf.GeneratedHCLs() {
+	for filename, genHCL := range stackHCLs.GeneratedHCLs() {
 		targetpath := filepath.Join(stackpath, filename)
 		logger := logger.With().
 			Str("blockName", filename).
 			Str("targetpath", targetpath).
 			Logger()
 
 		logger.Trace().Msg("Checking if code is updated.")
-
-		tfcode := prependGenHCLHeader(genHCL.Origin(), genHCL.String())
-		currentTfCode, err := loadGeneratedCode(targetpath)
+		currentHCLcode, codeFound, err := loadGeneratedCode(targetpath)
 		if err != nil {
 			return nil, err
 		}
+		if !codeFound && genHCL.String() == "" {
+			logger.Trace().Msg("Updated since file not found and generated_hcl is empty")
```

Comment:
shouldn't it be **not updated** instead?

Reply:
ahhh! I got it - it means that the file is not outdated, right? sorry for the noise